pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -393,7 +393,11 @@ void createNameSpace(String cluster, String publicTenant, NamespaceName ns) thro
         }
 
         if (!nsr.namespaceExists(ns)) {
-            broker.getAdminClient().namespaces().createNamespace(ns.toString());
+            try {
+                broker.getAdminClient().namespaces().createNamespace(ns.toString());
+            } catch (Exception e) {
+                log.warn(e.getMessage());
```

Comment:
```suggestion
                log.warn("...", e);
```

Besides, if the default manner to start a standalone cluster falls back to this branch, how can a standalone cluster successfully create the namespace (with which extra configs)?

Reply:
When the authentication/authorization is enabled, we must configure the `brokerClientAuthenticationPlugin` and `brokerClientAuthenticationParameters`.

Reply:
The namespace could be created via `pulsar-admin` with correct authentication info later. See how the C++ client's unit tests set up the test env. https://github.com/apache/pulsar-client-cpp/blob/2018a06e8afcde4de59971cfbc5f653a4f9d7897/build-support/start-test-service-inside-container.sh#L59-L63